apis/components/v1alpha1/codeflare_types.go

```diff
@@ -30,6 +30,7 @@ const (
 
 // CodeFlareCommonStatus defines the shared observed state of CodeFlare
 type CodeFlareCommonStatus struct {
+	Releases []common.ComponentReleaseStatus `json:"releases,omitempty"`
```

Comment:
it this is expected to be in reality like a map, maybe you can add some annotations like:

```
	// +patchMergeKey=name
	// +patchStrategy=merge
	// +listType=map
	// +listMapKey=type
```

Reply:
yep, but it is actually a map, like conditions i.e. you can't have multiple entries with the same name, I guess

Reply:
Oh, yes, that's true. Let me add the annotations.